main.go

```diff
@@ -10,18 +10,13 @@ import (
 )
 
 func main() {
-	exitCode := 0
-	defer func() {
-		os.Exit(exitCode)
-	}()
 
```

Comment:
You'll need to remove this empty line.

Reply:
Not sure why but line removed.

Reply:
Part of gofumpt. No empty lines after function declaration.

Reply:
```
No empty lines around function bodies
example

func foo() {

	println("bar")

}

func foo() {
	println("bar")
}
```

Reply:
Oh got it. It's removed anyway. I guess it's good now. Let me know if there is anything else that I should do here.